vuu-ui/showcase/src/examples/Layout/LayoutsBrowser.examples.tsx

```diff
@@ -0,0 +1,36 @@
+
+const mockLayoutsMetadata: LayoutMetadata[] = [
```

Comment:
This mock data doesn't seem to match what you have in your screenshot. Did you change it since taking the screenshot, or is the grouping not working how I'd expect?

Reply:
I have added this commit since taking the screenshot: https://github.com/ScottLogic/finos-vuu/pull/32/commits/fd8f7affdb5a38f80775e842785a0f5f8dd5eb27. I ll update the screenshot 👍